python/ray/serve/_private/config.py

```diff
@@ -147,6 +155,20 @@ def user_config_json_serializable(cls, v):
 
         return v
 
+    @validator("logging_config", always=True)
```

Comment:
should we also be validating that the encoding is a known value?

Reply:
Added. the DeploymentConfig is not user facing and got converted from Schema. So I am not too worried about the validation as long as we have strict check with the Schema.

Reply:
I imported schema here to handle the default value and then do type-checking. So that we don't need to have other class which will be duplicated with LoggingConfig in the schema.
(e.g. AutoscalingConfig in config.py and schema.py)